msp/factory.go

```diff
@@ -60,8 +60,8 @@ func New(opts NewOpts, cryptoProvider bccsp.BCCSP) (MSP, error) {
 			return newBccspMsp(MSPv1_3, cryptoProvider)
 		case MSPv1_4_3:
 			return newBccspMsp(MSPv1_4_3, cryptoProvider)
-		case MSPv1_4_4:
-			return newBccspMsp(MSPv1_4_4, cryptoProvider)
+		case MSPv2_4:
```

Comment:
We cannot put this in 2.4, since 2.4 already exists. 

I suggest we put this in Fabric v3.0.